README.md

```diff
@@ -10,16 +10,17 @@ in various languages and build an installer for it.
 
 ## Documentation
 
-[Citrix XenCenter Plug-in Specification Guide](https://developer-docs.citrix.com/projects/xencenter-plugin-specification/en/latest/)
+[XenCenter Plug-in Specification Guide](https://docs.citrix.com/en-us/citrix-hypervisor/developer/xencenter-plugin-specification.html)
 contains the complete specification for developing XenCenter plug-ins.
 
 A walk-through of the examples and instructions on how to compile the plug-in
 installers are available in the [docs](docs/README.md) pages.
 
 ## Maintainers
 
-* Mihaela Stoica <mihaela.stoica@citrix.com>
-* Konstantina Chremmou <konstantina.chremmou@citrix.com>
+* Konstantina Chremmou <konstantina.chremmou@cloud.com>
+* Danilo Del Busso <danilo.delbusso@cloud.com>
+* Chris Lancaster <chrislancaster@cloud.com>
```

Comment:
I checkd and we get no emails without a dot :( 

```suggestion
* Chris Lancaster <chris.lancaster@cloud.com>
```

Reply:
👀 No idea how I missed that!